<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QDialog" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-image:url(:/images/images/classGM/loginback.jpg)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/images/images/else/icon1.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电子教室教学管理软件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="loginW" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#loginW
{
	background:rgb(250, 250, 250)
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labFindPwd">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	color: rgb(0, 133, 199)
}
QLabel:hover
{
	color:  rgb(0, 170, 255)
}</string>
        </property>
        <property name="text">
         <string>找回密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>真实姓名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Algerian</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Adobe 仿宋 Std R</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账号：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>使用目的：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cbxPurpose">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox{border:1px solid #d7d7d7; border-radius: 3px; padding: 1px 18px 1px 3px;} 
QComboBox:editable{ background: white; }
QComboBox:!editable{ background: #fbfbfb; color:black}
QComboBox::drop-down{ subcontrol-origin: padding; subcontrol-position: top right; width: 22px; border-left-width: 1px; border-left-color: #c9c9c9; border-left-style: solid;  border-top-right-radius: 3px;border-bottom-right-radius: 3px; }
QComboBox::down-arrow:on {
top: 1px; left: 1px;
image:url(:/images/images/TiTleBar/setting_white_down.png);
}
QComboBox::down-arrow { 
image: url(:/images/images/TiTleBar/setting_white_normal.png);
}
QComboBox:hover{
border:2px solid rgb(50, 180, 250);

}
QComboBox:focus {
border:1px solid rgb(50, 180, 250);
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="leOtherPurpose">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>其他目的</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labmsg">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	color:red;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labmsgName">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	color:red;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>局域网ip设置</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>实验室及数据库设置</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
